src/test/resources/regressions/features/ghost_members/ghost-overflows.gobra

```diff
@@ -0,0 +1,15 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package main
+
+// constants have arbitrary precision, thus this declaration is valid:
+const large = 21342134123412341234132452314213
+
+func main() {
+    // non-ghost and ghost code should behave the same in terms of overflows:
```

Comment:
We can postpone this discussion but what is the reason for that? I may be missing something but I think that the last time this was mentioned, we agreed on having overflow checks for ghost code.
https://github.com/viperproject/gobra/pull/108#discussion_r515024797

Reply:
I should have written "that is not necessarily the case", I eddited that. My opinion is that there should be overflow checks in ghost code, but at one of our meetings we decided (for now) that there should not be overflow checks. That meeting happened after the post that you are referencing. Since, from my perspective, this discussion was not brought to a final conclusion, I am fine with just having overflow checks in ghost code for now.

Reply:
@ArquintL suggested another possible design which is to have a type for unbounded integers that can only be used in ghost code. That way, we can declare ghost variables of any kind supported in Gobra and also, declare "mathematical" integer variables. I am still unsure whether users would actually want to use bounded types in ghost code though and that is definitely one of the things we should have in mind when choosing one design

Reply:
To elaborate on @jcp19 comment: we discussed yesterday that it might be increasingly hard for a user to identify ghost code as the general direction is to infer more and more ghostness and reduce the burden to explicitly annotate something as being ghost. Thus, one argument could be that ghost and non-ghost code behaves the same by default (i.e. when they are syntactically the same). This would then imply that e.g. an explicit cast to an unbounded int would be necessary if you want it.